website/content/v1.3/introduction/system-requirements.md

```diff
@@ -52,4 +52,9 @@ description: "Hardware requirements for running Talos Linux."
   </tbody>
 </table>
 
-These requirements are similar to that of kubernetes.
+These requirements are similar to that of Kubernetes.
+## Storage
+Talos Linux itself only requires less than 100 MB of disk space.  
```

Comment:
should be at least 1 GiB minimum

but we should point out that EPHEMERAL is used to store pulled images, container work directories, so really meaningful minimum is 10 GiB

Something in the range 100-200 GiB makes more sense

Reply:
I think fixed this, but probably did it wrong. :-)